### PR TITLE
Python 3.5, 3.6 and PyPy support.

### PR DESCRIPTION
**Pull request by jamadden**
- Add a test for the configure.zcml and correct the errors exposed.
- Include zope.security when we test so the whole ZCML file gets included.
- Enable pip caching on Travis
- Enable coveralls on Travis.
- Fix long-description rendering error exposed by restview.

### CODE REVIEW

File: src/z3c/bcrypt/configure.zcml [hunk 1]
```diff
@@ -2,6 +2,10 @@
     xmlns="http://namespaces.zope.org/zope"
     xmlns:zcml="http://namespaces.zope.org/zcml">
 
+  <include package="zope.component" file="meta.zcml" />
+  <configure zcml:condition="installed zope.security">
+    <include package="zope.security" file="meta.zcml" />
+  </configure>
```
Comment (mgedmin) [inline src/z3c/bcrypt/configure.zcml:8]: FWIW I'm pretty sure you can do

```xml
<include package="zope.security" file="meta.zcml" zcml:condition="installed zope.security" />
```

**Review by mgedmin — APPROVED** [review]
LGTM

**Comment by jamadden** [comment]
Thanks! A PyPI release would be nice, but isn't holding anything up at this point.

**Comment by mgedmin** [comment]
> Package Index Owner: jw, janjaapdriessen

Pinging @janwijbrand and @janjaapdriessen

(Our PyPI usernames are `mgedmin` and `jamadden` if you're too busy to make a release and don't mind other people doing it.)

**Comment by janjaapdriessen** [comment]
Added `mgedmin` and `jamadden` as owners on pypi.

**Comment by jamadden** [comment]
Thank you! Released to PyPI.

**Pull request merged**